pages/pipelines/build_matrix.md

```diff
@@ -11,8 +11,7 @@ The following [command step](/docs/pipelines/command-step) attributes can contai
 * [plugins](/docs/pipelines/command-step#plugins)
 * [agents](/docs/pipelines/command-step#agents)
 
-You can't use matrix values in other attributes, including step keys, but you can use them in [concurrency groups](/docs/pipelines/controlling-concurrency#concurrency-groups).
-
+You can't use matrix values in other attributes, including step keys and concurrency groups.
```

Comment:
Was removing the link intentional?

I think it's probably best to keep it if not:

```suggestion
You can't use matrix values in other attributes, including step keys and [concurrency groups](/docs/pipelines/controlling-concurrency#concurrency-groups).
```

Reply:
Most likely oversight - I added it back @mbelton-buildkite 💚